### ISSUE TITLE
Cannot create encumbrances report

### ISSUE THREAD

**Issue by shelleydoljack** [sul-dlss/libsys-webforms#624]
I got an error message when trying to create an encumbrance report. Looks like another error with the `date_request` field. Here is from the rails log on dev:
```
Started GET "/assets/encumbrance_reports.self-7e6d99529499a7bdef26f183c7af084424b21fe9259cca326ed13c6062d16789.js?body=1" for 171.66.186.210 at 2019-09-23 15:15:28 -0700
Started POST "/encumbrance_reports" for 171.66.186.210 at 2019-09-23 15:16:08 -0700
Processing by EncumbranceReportsController#create as HTML
  Parameters: {"utf8"=>"✓", "authenticity_token"=>"REDACTED", "encumbrance_report"=>{"email"=>"sdoljack@stanford.edu", "fund_select"=>"fund_name_id", "fund"=>["REDACTED"], "fund_begin"=>"", "funding_paid"=>"More than $0 paid", "fundcyc_cycle"=>"2019", "status"=>"REQUEST", "output_file"=>"enc_rpt1569276928", "date_request"=>"", "date_ran"=>"", "message"=>""}, "commit"=>"Submit request"}
Completed 500 Internal Server Error in 8ms (ActiveRecord: 2.7ms)


  
ActiveRecord::StatementInvalid (OCIError: ORA-01830: date format picture ends before converting entire input string: INSERT INTO "ENCUMBRANCE_RPTS" ("DATE_REQUEST", "STATUS", "EMAIL", "FUND_ACCT", "FUNDCYC_CYCLE", "FUNDING_PAID", "OUTPUT_FILE") VALUES (:a1, :a2, :a3, :a4, :a5, :a6, :a7)):
  
app/controllers/encumbrance_reports_controller.rb:18:in `create'
```

**Comment by shelleydoljack** [comment]
Still failed. Here is the new failure:
```
Started GET "/assets/encumbrance_reports.self-7e6d99529499a7bdef26f183c7af084424b21fe9259cca326ed13c6062d16789.js?body=1" for 171.66.184.171 at 2019-09-24 15:53:33 -0700
Started POST "/encumbrance_reports" for 171.66.184.171 at 2019-09-24 15:54:44 -0700
Processing by EncumbranceReportsController#create as HTML
  Parameters: {"utf8"=>"✓", "authenticity_token"=>"REDACTED", "encumbrance_report"=>{"email"=>"sdoljack@stanford.edu", "fund_select"=>"fund_name_id", "fund"=>["REDACTED"], "fund_begin"=>"", "funding_paid"=>"More than $0 paid", "fundcyc_cycle"=>"2017", "status"=>"REQUEST", "output_file"=>"enc_rpt1569365613", "date_request"=>"2019-09-24 15:53:33", "date_ran"=>"", "message"=>""}, "commit"=>"Submit request"}
Completed 500 Internal Server Error in 6ms (ActiveRecord: 1.0ms)
  
ActiveRecord::StatementInvalid (OCIError: ORA-01830: date format picture ends before converting entire input string: INSERT INTO "ENCUMBRANCE_RPTS" ("DATE_REQUEST", "STATUS", "EMAIL", "FUND_ACCT", "FUNDCYC_CYCLE", "FUNDING_PAID", "OUTPUT_FILE") VALUES (:a1, :a2, :a3, :a4, :a5, :a6, :a7)):
```

**Comment by jgreben** [comment]
Fixed by 1fcb065

**Comment by shelleydoljack** [comment]
Confirming fix. I was able to create and received a report.
